### PR TITLE
feat: Add Helm3-compatible CRDs directory.

### LINKED ISSUE #196: Helm 3 compatibility for argo chart
**Is your feature request related to a problem? Please describe.**
#169 made the argo-cd chart compatible with both Helm 2 and Helm 3. I'd expect the same setup to work with the argo (workflows) chart.

**Additional context**
I'm happy to make the PR if there's agreement that it's worth doing 😄

### PR DESCRIPTION
**Pull request by danxmoran**
Resolves #196 

Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v2.4.3"
 description: A Helm chart for Argo Workflows
 name: argo
-version: 0.6.5
+version: 0.6.6
```
Comment (danxmoran) [inline charts/argo/Chart.yaml]: The Argo CD change was a patch bump, so I mirrored it here.

**Comment by evaneaston** [comment]
@benjaminws @danxmoran Any chance this can get some love and be merged?

**Comment by danxmoran** [comment]
I'm not sure how to fix the failing check, but I'll prioritize making any requested changes. My team is also blocked waiting for this to merge, we'll need to set up our own chart if it doesn't merge soon.

**Comment by evaneaston** [comment]
Perhaps change PR title to "feat: Add Helm3-compatible CRDs directory."

**Comment by peterloron** [comment]
All checks are passing now. @benjaminws Can we get an approval? Thanks!

**Comment by bclermont** [comment]
Please

**Comment by bclermont** [comment]
This pull request is outdated and the change to chart version don't make sense anymore, as since then the chart was updated to 1.6.x.

I just created PR #210 that upgrade Argo CD to 1.4.0 and rebased this PR on top of it and incremented the version.

**Comment by bclermont** [comment]
> This pull request is outdated and the change to chart version don't make sense anymore, as since then the chart was updated to 1.6.x.
> 
> I just created PR #210 that upgrade Argo CD to 1.4.0 and rebased this PR on top of it and incremented the version.


Well, even rebase the PR don't fix the problem with helm 3

```console
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(ConfigMap.data): invalid type for io.k8s.api.core.v1.ConfigMap.data: got "string", expected "map"
```

**Comment by paguos** [comment]
Any plans to merge this?

**Comment by alexec** [comment]
Can this be close?

**Comment by danxmoran** [comment]
Yes, looks like the same changes got merged in #229

**Pull request closed without merging**